tests/benchmarks/multicore/idle_ppr/testcase.yaml

```diff
@@ -0,0 +1,45 @@
+common:
+  sysbuild: true
+  tags: ci_build ci_tests_benchmarks_multicore
+  platform_allow:
+    - nrf54h20dk/nrf54h20/cpuapp
+  integration_platforms:
+    - nrf54h20dk/nrf54h20/cpuapp
+
+tests:
+  benchmarks.multicore.idle_ppr.no_sleep:
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
+      regex:
+        - "Multicore idle_ppr test on"
+        - "idle_ppr: Run 0"
+        - "idle_ppr: Run 1"
+
+  benchmarks.multicore.idle_ppr.idle:
+    tags: ppk_power_measure
+    extra_args:
+      idle_ppr_CONF_FILE=prj_s2ram.conf
+      remote_rad_CONF_FILE=prj_s2ram.conf
+      remote_ppr_CONF_FILE=prj_s2ram.conf
+      idle_ppr_CONFIG_TEST_SLEEP_DURATION_MS=500
+      remote_rad_CONFIG_TEST_SLEEP_DURATION_MS=500
+      remote_ppr_CONFIG_TEST_SLEEP_DURATION_MS=500
+    harness: pytest
+    harness_config:
+      fixture: ppk_power_measure
+      pytest_root:
+        - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_pwm_and_idle"
```

Comment:
pwm?

Reply:
This is temporary. When I confirm current consumption I will prepare dedicated pytest test.

Reply:
I see that a new pytest test is needed. Looks like time flows differently on each core thus blinking of three LEDs get unsynchronized in time.
As a result, active time lasts for 1.14 sec while expected is 1.0.

Reply:
New pytest check is needed. Time tolerance must be increased because cores run at different frequencies and don't enter sleep at exactly the same moment. See:
https://projecttools.nordicsemi.no/bitbucket/projects/NCS-TEST/repos/test-zephyr-sanitycheck/pull-requests/1547/overview